src/router.ts

```diff
@@ -1,9 +1,11 @@
 import Vue from 'vue';
```

Comment:
To enable proper code review, you HAVE TO remove the indendation changes from this PR or at least separate them out into a different commit, otherwise it's impossible to tell for us what changed and what hasn't.

Reply:
Just ignore whitespace: https://github.com/nimiq/accounts/pull/91/files?w=1